horton/gbasis/test/test_ints.py

```diff
@@ -2187,6 +2187,83 @@ def test_nuclear_attraction_co_ccpv5z_cart_hf():
     check_g09_nuclear_attraction(context.get_fn('test/co_ccpv5z_cart_hf_g03.fchk'))
 
 
+def check_g09_dipole(fn_fchk, dipole_values):
+    mol = IOData.from_file(fn_fchk)
+    xyz_array = np.array([[1,0,0], [0,1,0], [0,0,1]])
+    center = np.zeros(3)
+    dm_alpha = mol.exp_alpha.to_dm()
+    dipole = []
+    for xyz in xyz_array:
+        dipole_ints = mol.obasis.compute_multipole_moment(xyz, center, mol.lf)
+        ham = REffHam([RTwoIndexTerm(dipole_ints, 'dipole')])
+        ham.reset(dm_alpha)
+        dipole_v = -ham.compute_energy()
```

Comment:
The three lines above can be replaced by a single line without using `REffHam` and `RTwoIndexTerm`. Something along the following lines should work:

    dipole_v = -2*dipole_ints.contract_two('ab,ab', dm_alpha)

Alternatively you could also do

    dipole_v = -dipole_ints.contract_two('ab,ab', mol.dm_full)

Then you don't need to create `dm_alpha` and it would also work on open-shell cases with the same code.